addons/l10n_tr/data/account_tax_template_data.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <!-- account.tax.template -->
-    <record id="tr_kdv_satis_sale_18" model="account.tax.template">
+    <record id="tr_kdv_satis_sale_20" model="account.tax.template">
```

Comment:
@dbkosky in case if you missed, am not sure if it is allowed to change external id of a record in stable, should we keep this tax as it and create a new one for 20%

Reply:
:man_facepalming:  yep that was an oversight
we can set "active"=False on the existing tax